pepper-apis/auth0-config/rules/register-user-in-pepper.js

```diff
@@ -95,6 +95,10 @@ function (user, context, callback) {
             console.log('No Registration Mode passed in request');
         }
 
+        if (context.request.body.invitation_id) {
```

Comment:
It seems web clients pass these in the query params instead of body. Let's check `context.request.query` as well.